salt/etcd/init.sls

```diff
@@ -32,10 +32,6 @@ etcd:
       - etcd
     - require:
       - file: /etc/zypp/repos.d/containers.repo
-  cmd.run:
-    - name: rm -rf /var/lib/etcd/*
-    - prereq:
-      - service: etcd
```

Comment:
Yea, that scares me. Maybe it should be split off and proposed separately to master?